config/routes.rb

```diff
@@ -14,6 +15,8 @@
   resources :categories, only: [:index, :show]
   resources :listings, only: [:index]
 
+  post '/noteifications' => 'notification#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.